flake.nix

```diff
@@ -69,10 +71,12 @@
             buildInputs =
               [
                 cairo
+                curl
```

Comment:
This other [curl based thing](https://github.com/NixOS/nixpkgs/blob/dd37924974b9202f8226ed5d74a252a9785aedf8/pkgs/applications/networking/flent/http-getter.nix#L15) uses `curl` as well. So this seems fine.